### PR TITLE
Problem: v8.0.0-gravity-alpha1 upgrade is not tested

### PR DESCRIPTION
**Pull request by thomas-nguy**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

for review only

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

**Review by yihuang — APPROVED** [review]

File: app/app.go [hunk 1]
```diff
@@ -742,12 +742,39 @@ func New(
 		return app.mm.RunMigrations(ctx, app.configurator, fromVM)
 	})
 
+	gravityPlanName := "v0.8.0-gravity-alpha1"
+	app.UpgradeKeeper.SetUpgradeHandler(gravityPlanName, func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
```
Comment (yihuang) [inline app/app.go:746]: we can even put this logic under the condition `if experimental {`

Comment (thomas-nguy) [reply app/app.go]: added

File: app/app.go [hunk 2]
```diff
@@ -739,12 +739,41 @@ func New(
...
+			// register new params
+			gravParamStore := app.GetSubspace(gravitytypes.ModuleName)
+			gravParamStore.Set(ctx, gravitytypes.ParamStoreBridgeActive, true)
+			gravParamStore.Set(ctx, gravitytypes.ParamStoreBatchCreationPeriod, uint64(10))
+			gravParamStore.Set(ctx, gravitytypes.ParamStoreBatchMaxElement, uint64(100))
+			gravParamStore.Set(ctx, gravitytypes.ParamStoreObserveEthereumHeightPeriod, uint64(50))
+
+			// set new gravity id
+			gravParams := app.GravityKeeper.GetParams(ctx)
+			gravParams.GravityId = "cronos_gravity_pioneer_v2"
```
Comment (thomas-nguy) [inline app/app.go:758]: For info @allthatjazzleo @borischeuk-crypto this is the new gravity_id

File: app/app.go [hunk 3]
```diff
@@ -739,12 +739,41 @@ func New(
...
+			app.GravityKeeper.SetParams(ctx, gravParams)
+
+			// Estimate time upgrade take place
+			// 100% is not necessary here because it will be tuned by relayers later on
+			// it is set to georli height at 23th August 2022 3pm JST
+			app.GravityKeeper.MigrateGravityContract(
```
Comment (thomas-nguy) [inline app/app.go:764]: for info @lezzokafka , upgrade day should be after 23th August 2022 3pm JST so 24th should be okay

**Pull request merged**